src/common/get-file-info.js

```diff
@@ -19,6 +19,14 @@ const path = require("path");
  * internally by the method wrapper. See withPlugins() in index.js.
  */
 function getFileInfo(filePath, opts) {
+  if (typeof filePath !== "string") {
+    return Promise.reject(
```

Comment:
Just use `throw new TypeError`, no need use `Promise.reject`

Reply:
no, that would  be a breaking  change since thi function I not async

Reply:
my mistake (i think it is async :smile: )